raiden_installer/ethereum_rpc.py

```diff
@@ -35,14 +35,15 @@ def gas_price_strategy_eth_gas_station_or_with_margin(web3: Web3, transaction_pa
         # FIXME: This is a temporary fix to speed up gas price generation
         # by fetching from eth_gas_station if possible.
         # Once we have a reliable gas price calculation this can be removed
-        try:
-            response = requests.get(ETH_GAS_STATION_API)
-            if response and response.status_code == 200:
-                data = response.json()
-                log.debug(f"fetched gas price: {Wei(int(data['fast'] * 10e7 * 1.1))} Wei")
-                return Wei(int(data["fast"] * 10e7 * 1.1))
-        except (TimeoutError, ConnectionError, KeyError):
-            log.debug("Could not fetch from ethgasstation. Falling back to web3 gas estimation.")
+        if int(web3.net.version) == 1:
```

Comment:
Because we don't want to use the gas prices from eth gas station on the testnets. The goerli wizard used way too much gas for the transactions before.